Makefile

```diff
@@ -1,3 +1,6 @@
+run:
+	node_modules/.bin/web-ext run
+
```

Comment:
Things that have no real dependencies should be marked as phony. Also keeping the `old-reddit-redirect.zip` entry first ensures a plain `make` builds the app. So this file should just be:

```
.PHONY: run clean

old-reddit-redirect.zip: *.json *.js img/* *.md *.txt
	zip -r old-reddit-redirect.zip * -x .git/* -x img/screenshot.png -x .gitignore -x Makefile

run:
	npx web-ext run

clean:
	rm -f *.zip
```